### DISCUSSION TITLE
workaround request to add tables to repset if they dont have primary key

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#31] [Q&A]
Hi guys, 

I have few tables without primary_key and when i try to add them to repset i had created it fails with below error. Can you please advise how to sort this out? thanks

Adding table `schema.abcdtable` to replication set `abc_replication_set`.
WARNING: table `abcdtable` cannot be added to replication set `abc_replication_set`
WARNING: DETAIL:  table does not have PRIMARY KEY and given replication set is configured to replicate UPDATEs and/or DELETEs

postgres v15.3

**Comment by luss** [comment]
You can manually add a table without a primary key using repset-add-table.   Tables without a primary key cannot replicate update and delete statements

**Reply by luss** [reply]
We also now support an automatic mode that does this for u
